askama_shared/src/io/mod.rs

```diff
@@ -0,0 +1,36 @@
+
+pub struct WriteIoToFmt<W : std::io::Write> {
+	write : W,
+	error : Option<std::io::Error>,
+}
+
+
+impl <W : std::io::Write> WriteIoToFmt<W> {
+	
+	pub fn new(write: W) -> Self {
+		return Self{write: write, error: None};
+	}
+	
+	pub fn done(self) -> Option<std::io::Error> {
```

Comment:
Let's just call this `error()`.

Reply:
OK, I've solved it.  (Will be in next push.)